components/account/home/SubscriptionTable.vue

```diff
@@ -116,17 +116,19 @@ watch(pending, (pending) => {
 
 onUnmounted(() => pause());
 
-function hasAction(sub: Subscription, action: 'renew' | 'cancel') {
+function hasAction(sub: Subscription, action: 'renew' | 'cancel' | 'continue') {
   if (action === 'cancel')
     return sub.status !== 'Pending' && sub.actions.includes('cancel');
   else if (action === 'renew')
     return sub.actions.includes('renew');
+  else if (action === 'continue')
```

Comment:
Is backend supposed to return continue as an action?
I am trying to replicate this to test but I don't seem to be able.

Reply:
so the action there is not checked against the status on the backend, but I create my own status in frontend.
But yeah it's confusing, I will update it, create new function `isPending` instead.